tests/httpServer/adminserver.test.ts

```diff
@@ -18,15 +18,32 @@ describe('not-running-admin-server', () => {
   test('test-admin-server-not-running', async () => {
     config = generatePublicDBOSTestConfig({ runAdminServer: false });
     DBOS.setConfig(config);
-    const [translatedConfig] = translatePublicDBOSconfig(config);
-    await setUpDBOSTestDb(translatedConfig);
+    await setUpDBOSTestDb(config);
     await DBOS.launch();
 
     await expect(async () => {
       await fetch(`http://localhost:3001${HealthUrl}`, {
         method: 'GET',
       });
     }).rejects.toThrow();
+
+    await DBOS.shutdown();
+  });
+
+  test('admin-port-already-in-use', async () => {
+    // Start a dummy server on the admin port
+    const server = http.createServer().listen(3001, '127.0.0.1');
+    try {
+      config = generatePublicDBOSTestConfig({ runAdminServer: true });
+      DBOS.setConfig(config);
+      await setUpDBOSTestDb(config);
+      await DBOS.launch();
+      await DBOS.shutdown();
+    } catch (error) {
+      expect((error as Error).message).toMatch(/EADDRINUSE/);
```

Comment:
Isn't the whole point that we don't throw?

Reply:
ha good catch, we actually don't throw when starting, but somehow this catch would still surface the exception. I removed it because it is misleading.